build.gradle

```diff
@@ -68,7 +68,7 @@ dependencies {
     library "org.reflections:reflections:${project.reflections_version}"
     library("io.netty:netty-handler-proxy:${project.netty_version}") { transitive = false }
     library("io.netty:netty-codec-socks:${project.netty_version}") { transitive = false }
-    include(implementation("com.github.LlamaLad7.MixinExtras:mixinextras-fabric:${project.mixin_extras_version}"))
+    library "com.github.LlamaLad7.MixinExtras:mixinextras-fabric:${project.mixin_extras_version}"
```

Comment:
Use ModInclude instead of library is better I think

Reply:
Minegame recommended to use library, to keep consistencey with the rest of them. Is modInclude different or better to library?